pylxd/models/container.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2016 Canonical Ltd
+# Copyright (c) 2020 Canonical Ltd
```

Comment:
This happened because I did git mv container.py instance.py and then created a new container.py. In MAAS we always update the copyright year to the last time it was touched so I've changed this to 2016-2020. Let me know if it should just be 2016.